src/rules/javascript/webextension-unsupported-api.js

```diff
@@ -5,7 +5,8 @@ export default {
   create(context) {
     return {
       MemberExpression: function(node) {
-        if (node.object.object &&
+        if (!node.computed &&
```

Comment:
Since we rely more and more on auto-detection, I'm not sure it's a good idea to just skip them.

Do you think this would lead to a lot of real false positives?